src/main/java/ch/njol/skript/bukkitutil/EnchantmentUtils.java

```diff
@@ -18,26 +18,106 @@
  */
 package ch.njol.skript.bukkitutil;
 
+import ch.njol.skript.localization.Language;
 import org.bukkit.NamespacedKey;
+import org.bukkit.Registry;
 import org.bukkit.enchantments.Enchantment;
 import org.eclipse.jdt.annotation.Nullable;
 
-import com.google.common.collect.BiMap;
-import com.google.common.collect.HashBiMap;
 import ch.njol.skript.Skript;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
 /**
  * Maps enchantments to their keys.
  */
 public class EnchantmentUtils {
 
+	private static final Map<Enchantment, String> NAMES = new HashMap<>();
+	private static final Map<String, Enchantment> PATTERNS = new HashMap<>();
+	private static final boolean HAS_REGISTRY = Skript.classExists("org.bukkit.Registry") && Skript.fieldExists(Registry.class, "ENCHANTMENT");
+
+	static {
+		Language.addListener(() -> {
+			NAMES.clear();
+			PATTERNS.clear();
+			List<Enchantment> enchantments = new ArrayList<>();
+			if (HAS_REGISTRY) {
+				Registry.ENCHANTMENT.forEach(enchantments::add);
+			} else {
+				enchantments.addAll(Arrays.asList(Enchantment.values()));
+			}
+			for (Enchantment enchantment : enchantments) {
```

Comment:
For consistency (and to avoid duplicating behavior), maybe we should take a similar approach to what we did with Biomes. That is, we can keep the previous behavior if HAS_REGISTRY is false, but when it's true, let's just create/use a RegistryParser.

Reply:
I won't be able to do that until the registry PR is merged.